doc/usr/howto/installation.md

```diff
@@ -47,7 +47,7 @@ building and running the complete `nope.gl` stack.
   `C:\vcpkg`, then from Windows PowerShell:
     ```shell
     .\bootstrap-vcpkg.bat
-    .\vcpkg.exe install --triplet x64-windows opengl-registry ffmpeg[ffmpeg,ffprobe,zlib] sdl2 glslang
+    .\vcpkg.exe install --triplet x64-windows ffmpeg[ffmpeg,ffprobe,zlib] sdl2 glslang
```

Comment:
We need a few words to explain to the user what they're expected to do then

Reply:
About what ? opengl-registry is pulled by `configure.py` automatically. The user does not need to do anything about it.

Reply:
Oh yeah no right, I'm an idiot, disregard that